commons/src/main/java/org/apache/mesos/elasticsearch/common/zookeeper/ZookeeperCLIParameter.java

```diff
@@ -20,4 +20,12 @@ public String getZookeeperUrl() {
     public long getZookeeperTimeout() {
         return zookeeperTimeout;
     }
+
+    public static final String ZOOKEEPER_FRAMEWORK_URL = "--zookeeperFrameworkUrl";
```

Comment:
I'm still not sure about the names --zookeeperURL and --zookeeperFrameworkUrl.
It makes me feel confused as to what zookeeperURL actually sets.
But I don't know what else to call it. Maybe --zookeeperDefaultURL? Or something? What do you think?